sdk/spring/azure-spring-boot-starter/README.md

```diff
@@ -9,6 +11,23 @@ This repository supports Spring Boot 2.1.x, 2.2.x and 2.3.x. Please read [Spring
 - JDK 1.8 and above
 - [Maven](http://maven.apache.org/) 3.0 and above
 
+### Include the Package
+
+`azure-spring-boot-starter` is published on Maven Central Repository.  
+If you are using Maven, add the following dependency.  
+
+[//]: # ({x-version-update-start;com.microsoft.azure:azure-spring-boot-starter;current})
+```xml
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-spring-boot-starter</artifactId>
+    <version>2.3.3-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+## Examples
+
 ## Key concepts
 ### Usage
 
```

Comment:
Should we mark the 2.3.x version for storage, servicebus, and mediaservices as deprecated? Which makes it more straightforward for the users.

Reply:
Marked in the Key concepts section.